.github/workflows/test_sharktank_models.yml

```diff
@@ -0,0 +1,72 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: Test Sharktank Models
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - ".github/workflows/test_sharktank_models.yml"
+      - "sharktank_models/**"
+  pull_request:
+    paths:
+      - ".github/workflows/test_sharktank_models.yml"
+      - "sharktank_models/**"
+  workflow_dispatch:
+  schedule:
+    # Runs at 3:00 PM UTC, which is 8:00 AM PST
+    - cron: "0 15 * * *"
+
+concurrency:
+  # A PR number if a pull request and otherwise the commit hash. This cancels
+  # queued and in-progress runs for the same PR (presubmit) or commit
+  # (postsubmit). The workflow name is prepended to avoid conflicts between
+  # different workflows.
+  group: ${{ github.workflow }}-${{ github.event.number || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  test-sharktank-models:
+    runs-on: ubuntu-24.04
+    env:
+      VENV_DIR: ${{ github.workspace }}/.venv
+      HTML_REPORT_PATH: sharktank_models/sharktank_models_test_report_cpu_llvm_task.html
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          lfs: true
+
+      # Install Python packages.
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
+      - name: Setup Python venv
+        run: python3 -m venv ${VENV_DIR}
+      - name: Install IREE nightly release Python packages
+        run: |
+          source ${VENV_DIR}/bin/activate
+          python3 -m pip install -r sharktank_models/requirements-iree.txt
+
+      # Run tests.
+      - name: Run Sharktank models test suite
+        run: |
+          source ${VENV_DIR}/bin/activate
+          pytest sharktank_models/ \
+            -rA \
+            -m "target_cpu" \
+            --timeout=300 \
+            --durations=0 \
+            --html=${HTML_REPORT_PATH} \
+            --self-contained-html
```

Comment:
FYI this contains no log output right now: 
![image](https://github.com/user-attachments/assets/81fac4b2-5a41-44be-9127-c0164531c33e)

Might get some info with `--log-cli-level=info` or the other flags (https://docs.pytest.org/en/stable/how-to/logging.html)

Reply:
Fixed

Reply:
Thanks!
```
------------------------------ Captured log setup ------------------------------
INFO     root:binaries.py:185 Invoke IREE Tool: /home/runner/work/iree-test-suites/iree-test-suites/.venv/lib/python3.11/site-packages/iree/compiler/tools/../_mlir_libs/iree-compile /home/runner/work/iree-test-suites/iree-test-suites/sharktank_models/llama3.1/assets/toy_llama.mlir --iree-input-type=auto --iree-vm-bytecode-module-output-format=flatbuffer-binary --mlir-print-debuginfo --mlir-print-op-on-diagnostic=false --iree-hal-target-device=llvm-cpu --iree-llvmcpu-target-cpu=host
```


![image](https://github.com/user-attachments/assets/7d7fe977-7d10-45a1-9f85-29b493986fd5)